src/triggers.js

```diff
@@ -266,6 +284,20 @@ export function getResponseObject(request, resolve, reject) {
       return resolve(response);
     },
     error: function(error) {
+      // handle special readOnlyTriggers cases
+      if (isReadOnlyTrigger) {
+        // Ignore thrown errors in beforeDelete & during login.
+        // We should prevent login from breaking if an error is thrown during the process,
```

Comment:
If it’s a readOnlyTrigger, why would we perform additional checks?

If the trigger is a simple event based one errors should simply get ignored. So it should resolved always.

Reply:
I want to give the possibility to reject a login, rejecting all readOnlyTrigger events would get in the way of that.
I am thinking now of refactoring the name to sessionTrigger rather than readonlyTrigger, it seems more appropriate.